SDK-1204 Updated docs (#686)
box/box-ios-sdk: docs/usage/files.md

To retrieve a list of collaborations on a file, call
[`client.files.listCollaborations(forFile:marker:limit:fields:)`][get-collaborations]

> with the ID of the file.  This method returns an iterator in the completion that can be used to retrieve file collaborations.

Reviewer: ```suggestion
with the ID of the file.  This method returns an iterator in the completion, which is used to retrieve file collaborations.
```

To retrieve a list of comments on the file, call
[`client.files.listComments(forFile:offset:limit:fields:)`][get-comments]

> with the ID of the file.  This method returns an iterator in the completion that can be used to page through the collection

Reviewer: ```suggestion
with the ID of the file.  This method returns an iterator in the completion, which is used to page through the collection
```

--------------

To retrieve a list of file tasks, call [`client.files.listTasks(forFile:fields:)`][get-tasks] with the ID of the

> file.  This method returns an iterator in the completion that can be used to retrieve file comments.

Reviewer: ```suggestion
file.  This method returns an iterator in the completion, which is used to retrieve file comments.
```
